### DISCUSSION TITLE
Mobile Number and OTP validation in Login

### DISCUSSION THREAD

**Discussion by sabarinathancs** [codeigniter4/shield#991] [Q&A]
I was planning to have inside of Email and Password, I would like to have Mobile Number and OTP option for Login Process, Will Shield has can help on this

**Comment by kenjis** [comment]
Yes, if you can customize Shield.
See https://shield.codeigniter.com/customization/adding_attributes_to_users/
https://shield.codeigniter.com/quick_start_guide/using_session_auth/#enable-two-factor-authentication
